docs/gcloud-auth.rst

```diff
@@ -58,15 +58,68 @@ properties of your local environment in the following order
 #. Credentials provided by the Google Cloud SDK (via ``gcloud auth login``)
 #. Application running in Google Compute Engine
 
-Loading Credentials Explicitly
-------------------------------
+Explicit Credentials
+====================
 
-In addition, the
+The Application Default Credentials discussed above can be useful if your
```

Comment:
I saw you suggest this on the Code of Conduct PR and find it to be a nebulous idea. I'll try my best.

Keep in mind, I already keep lines under 80 (unless it's a long link).